src/create-config.ts

```diff
@@ -358,7 +360,7 @@ const main = async (operationRepositoryTarget?: string) => {
       ...templateObj,
       [template.templateId]: {
         endpointId: template.endpointId,
-        templateParameters: template.parameters,
+        encodedParameters: template.parameters,
```

Comment:
I'm guessing we will need to merge the Airkeeper dependencies update [PR](https://github.com/api3dao/airkeeper/pull/64) to main. I was hoping to get a second review from Michal when he's back but Vekil has already approved it.